### DISCUSSION TITLE
Change Icon Text

### DISCUSSION THREAD

**Discussion by mwiikned** [weather-gov/api#375] [Q&A]
The old forecast-weather API that this is intended to replace has icons with green text and a white background. This new one switched to blue text on a slightly darker background and smaller font size.  This has made it so the icons need to be scaled up significantly to make the text clear. No idea if this is something that can be changed but more readable text would be nice on icons that have text.

**Comment by StephenClouse** [comment]
We do not plan to make any further changes to the icons in API. Our current intent is to deprecate and remove the icon functionality from the API at an undetermined future date.

Having these images served by API has never quite felt right, as it is the only presentational aspect of an otherwise entirely data service. It was done out of necessity to support the new version of forecast.weather.gov in a timely manner, but with that release on hold, we have had a chance to revisit some of these older decisions, and we have determined that the choice of icons should be the role of the consuming application, not API.

We plan to add richer data to the forecast output to allow consuming applications to implement their own icon logic.

**Reply by long-nguyen** [reply]
@StephenClouse 
Hello StephenClouse, I want to implement my own icon logic for my application, but I can not find any way to convert the result in this api (https://api.weather.gov/gridpoints/BOU/62,61/forecast) to icon.
May I use  https://api.weather.gov/icons to get all icon type, is that possible?
